build/__phpVersions.sh

```diff
@@ -28,4 +28,4 @@ PHP70_KEYS='1A4E8B7277C42E53DBA9C7B9BCAA30EA9C0D5763 6E4F6AB321FDC07F2C332E3AC2B
 PHP70_TAR_SHA256='ab8c5be6e32b1f8d032909dedaaaa4bbb1a209e519abb01a52ce3914f9a13d96'
 PHP56_VERSION='5.6.40'
 PHP56_KEYS='0BD78B5F97500D450838F95DFE857D9A90D90EC1 6E4F6AB321FDC07F2C332E3AC2BF0BC433CFC8B3'
-PHP56_TAR_SHA256='1369a51eee3995d7fbd1c5342e5cc917760e276d561595b6052b21ace2656d1c'
\ No newline at end of file
+PHP56_TAR_SHA256='1369a51eee3995d7fbd1c5342e5cc917760e276d561595b6052b21ace2656d1c'
```

Comment:
Could you please validate that it is expected that this SHA remains the same? Shouldn't the SHA change given the new version?

Reply:
This sha is for php5.6 so it's not expected to change, since we're only updating php-composer for this PR.